xml/System.IO/UnmanagedMemoryStream.xml

```diff
@@ -1574,10 +1574,10 @@
         <Parameter Name="cancellationToken" Type="System.Threading.CancellationToken" Index="1" FrameworkAlternate="netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="buffer">To be added.</param>
-        <param name="cancellationToken">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="buffer">Buffer that will be written.</param>
+        <param name="cancellationToken">Token that can be used to cancel the operation.</param>
+        <summary>Writes buffer into the stream. The operation completes synchronously.</summary>
+        <returns>A value task.</returns>
         <remarks>To be added.</remarks>
```

Comment:
```suggestion
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 If an exception occurs during the write operation, it will be set as the <xref:System.Threading.Tasks.Task.Exception%2A?displayProperty=nameWithType> of the property of the returned task.  
  
 ]]></format>
        </remarks>
```